test/repositories/slick/implementations/ChatsRepositorySpec.scala

```diff
@@ -526,4 +526,33 @@ class ChatsRepositorySpec extends AsyncWordSpec with MustMatchers with BeforeAnd
 
   }
 
+  "SlickChatsRepository#moveChatToTrash" should {
+    val chatsRep = new SlickChatsRepository(db)
+
+    val userId = "148a3b1b-8326-466d-8c27-1bd09b8378f3" //beatriz@mail.com
+
+    "remove the user's chat from inbox, sent and draft and move it to trash" in {
+      val validChatId = "303c2b72-304e-4bac-84d7-385acb64a616"
+      for {
+        result <- chatsRep.moveChatToTrash(validChatId, userId)
+        optionUserChat <- db.run(UserChatsTable.all.filter(uc => uc.chatId === validChatId && uc.userId === userId).result.headOption)
+        userChat = optionUserChat.get
+      } yield assert(
+        result &&
+          userChat.inbox === 0 &&
```

Comment:
What is the "===" method?

Reply:
Found a good explaination:
"ScalaTest lets you use Scala's assertion syntax, but defines a triple equals operator (===) to give you better error messages. The following code would give you an error indicating only that an assertion failed:
assert(1 == 2) 
Using triple equals instead would give you the more informative error message, "1 did not equal 2":
assert(1 === 2)"